pkg/collector/corechecks/cluster/orchestrator/orchestrator_transformer_test.go

```diff
@@ -42,6 +43,51 @@ func strPtr(v string) *string {
 	return &v
 }
 
+func getTemplateWithResourceRequirements() corev1.PodTemplateSpec {
```

Comment:
Do you test the template that only have resource limits or only have resource requests ?

Reply:
I currently have one test for this, as we already have those tests in the `podSpec`. But for sure I can add one 👍

Reply:
Example tests are here: https://github.com/DataDog/datadog-agent/blob/b39062e76cb14ea02cdb2e1a70770c3c8de2f189/pkg/util/orchestrator/pod_test.go#L450-L463